draft-irtf-cfrg-vdaf.md

```diff
@@ -4580,23 +4576,25 @@ are defined in {{poplar1-param}}. The methods required for sharding,
 preparation, aggregation, and unsharding are described in the remaining
 subsections. These methods make use of constants defined in {{poplar1-const}}.
 
-| Parameter         | Value                                |
-|:------------------|:-------------------------------------|
-| `VERIFY_KEY_SIZE` | `Xof.SEED_SIZE`                      |
-| `RAND_SIZE`       | `Xof.SEED_SIZE * 3 + Idpf.RAND_SIZE` |
-| `NONCE_SIZE`      | `16`                                 |
-| `ROUNDS`          | `2`                                  |
-| `SHARES`          | `2`                                  |
-| `Measurement`     | `tuple[bool, ...]`                   |
-| `AggParam`        | `tuple[int, Sequence[tuple[bool, ...]]]` |
-| `PublicShare`     | same as the IDPF                     |
+| Parameter         | Value                                      |
+|:------------------|:-------------------------------------------|
+| `VERIFY_KEY_SIZE` | `xof.SEED_SIZE`                            |
+| `RAND_SIZE`       | `xof.SEED_SIZE * 3 + idpf.RAND_SIZE`       |
+| `NONCE_SIZE`      | `16`                                       |
+| `ROUNDS`          | `2`                                        |
+| `SHARES`          | `2`                                        |
+| `Measurement`     | `tuple[bool, ...]`                         |
+| `AggParam`        | `tuple[int, Sequence[tuple[bool, ...]]]`   |
+| `PublicShare`     | same as the IDPF                           |
 | `InputShare`      | `tuple[bytes, bytes, list[FieldInner], list[FieldLeaf]]` |
-| `OutShare`        | `FieldVec`                           |
-| `AggShare`        | `FieldVec`                           |
-| `AggResult`       | `list[int]`                          |
-| `PrepState`       | `tuple[bytes, int, FieldVec]`        |
-| `PrepShare`       | `FieldVec`                           |
-| `PrepMessage`     | `Optional[FieldVec]`                 |
+| `OutShare`        | `FieldVec`                                 |
+| `AggShare`        | `FieldVec`                                 |
+| `AggResult`       | `list[int]`                                |
+| `PrepState`       | `tuple[bytes, int, FieldVec]`              |
+| `PrepShare`       | `FieldVec`                                 |
+| `PrepMessage`     | `Optional[FieldVec]`                       |
+| `xof`             | `XofTurboShake128` ({{xof-turboshake128}}) |
+| `idpf`            | as specified in {{idpf-bbcggi21}}          |
```

Comment:
We could similarly make the first paragraph of section 8.2 more concrete, i.e. replace "any IDPF for which SHARES == 2 and VALUE_LEN == 2" with "the IDPF from {{idpf-bbcggi21}}, with VALUE_LEN = 2 and ...".